cmd/argo/commands/submit.go

```diff
@@ -102,6 +102,10 @@ func SubmitWorkflows(filePaths []string, submitOpts *util.SubmitOpts, cliOpts *c
 		}
 	}
 
+	if !checkWfsInSameNs(workflows) {
+		log.Fatal("Workflows aren't at the same namespace")
```

Comment:
Or should we rather create workflows of different namespaces one by one by loop?

Reply:
That's a good idea.

But for the below logic,   like `watch`  maybe will have some confuse.   And check all the workflow are in the same ns. will help users to avoid mistakes on yaml file.